app/core/models/ocelot/stanzas/SequenceStanza.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package core.models.ocelot.stanzas
+
+import core.models.ocelot.{labelReferences, ScalarLabel, Labels, Label, Phrase, asListOfInt}
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Reads._
+import play.api.libs.json.{JsSuccess, JsError, JsValue, JsonValidationError, JsPath, OWrites, Reads}
+
+case class SequenceStanza(text: Int,
+                          override val next: Seq[String],
+                          options: Seq[Int],
+                          label: Option[String],
+                          stack: Boolean) extends VisualStanza
+
+object SequenceStanza {
+  implicit val reads: Reads[SequenceStanza] = (js: JsValue) =>
+    ((js \ "text").validate[Int] and
+      (js \ "next").validate[Seq[String]](minLength[Seq[String]](1)) and
```

Comment:
Do you have to define more than one flow in Ocelot?

Reply:
At least one

Reply:
Ohh it should be two, will fix